src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +181,98 @@ void draw_game_selector(GuiState &gui, HostState &host) {
+                        if (gui.delete_game_popup)
+                            gui.delete_game_popup = false;
+                    }
+
+                    if (gui.delete_game_popup && fs::exists(game_path / gui.title_id))
+                        ImGui::OpenPopup("Delete Game");
+                    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+                    if (ImGui::BeginPopupModal("Delete Game", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
+                        ImGui::PopStyleColor();
+                        ImGui::Text("Do you really want delete this game ?\n%s [%s]", game.title_id.c_str(), game.title.c_str());
+                        ImGui::Separator();
+                        if (ImGui::Button("Yes", ImVec2(124, 0))) {
+                            if (fs::exists(game_path / game.title_id))
+                                fs::remove_all(game_path / game.title_id);
+                            gui.game_selector.games.erase(games_index);
+                            if (gui.user_backgrounds[host.cfg.background_image] && gui.current_background != static_cast<std::uint32_t>(gui.user_backgrounds[host.cfg.background_image]))
+                                gui.current_background = gui.user_backgrounds[host.cfg.background_image];
+                            else if (!gui.user_backgrounds[host.cfg.background_image] && gui.current_background != 0)
+                                gui.current_background = 0;
+                            gui.delete_game_popup = false;
+                            ImGui::CloseCurrentPopup();
+                        }
+                        ImGui::SameLine();
+                        if (ImGui::Button("No", ImVec2(124, 0))) {
+                            gui.delete_game_popup = false;
+                            ImGui::CloseCurrentPopup();
+                        }
+                        ImGui::EndPopup();
+                    } else
+                        ImGui::PopStyleColor();
+                }
+
             }
             ImGui::NextColumn();
-            ImGui::Selectable(game.title_id.c_str(), &selected[1], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, icon_size));
+            ImGui::Selectable(game.title_id.c_str(), &selected[1], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, static_cast<float>(icon_size)));
+            if (ImGui::IsItemHovered() && gui.title_id != game.title_id)
+                gui.title_id = game.title_id;
+            ImGui::OpenPopupOnItemClick("#game_context_menu", 1);
             ImGui::NextColumn();
-            ImGui::Selectable(game.app_ver.c_str(), &selected[2], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, icon_size));
+            ImGui::Selectable(game.app_ver.c_str(), &selected[2], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, static_cast<float>(icon_size)));
             ImGui::NextColumn();
             if (ImGui::IsItemHovered()) {
-                if (host.cfg.show_game_background) {
```

Comment:
Why removing the config

Reply:
because i have just tested on things and can causing probleme for user disable game background after have set one game backgroud on font, so i have removed because not needed, i have set this me first time butr finnaly is not needed